src/store/index.ts

```diff
@@ -18,7 +19,7 @@ const state: any = {
 }
 
 const persistState = createPersistedState({
-    paths: ['user'],
+    paths: ['user', 'picklist', 'product'],
```

Comment:
Why we are persisting the picklist? Let say if the picklist are already processed by some other user, it will still appear on reopening the app from storage

Reply:
When I am removing the picklist from persist state. The products are not getting displayed after refreshing the page. That is why I have put picklist in persist state.

Reply:
I think this is not the right fix. @ymaheshwari1  has fixed it by adding it to updated lifecycle hook. Please verify and remove this

Reply:
Verified with Yash sir and removed it.